src/helix.idea.vim

```diff
@@ -0,0 +1,198 @@
+"""""""""""""""""""""""""
+" IdeaVim support       "
+"""""""""""""""""""""""""
+set nohlsearch                    " no highlight search results
+set number                        " line numbers
+set showmode                      " show current mode
+set gdefault                      " substitute all occurrences in line per default
+set history=10000                 " keep x lines of command line history
+set scrolloff=5 sidescrolloff=10  " keep some lines before and after the cursor visible
+set visualbell                    " no beep
+set smartcase                     " no ignore case when pattern is uppercase
+set incsearch                     " show search results while typing
+set wrapscan                      " searches wrap around the end of the file
+
+set ideajoin
+set idearefactormode=keep
+
+" Plugins
+set commentary
+set highlightedyank
+set argtextobj
+set matchit
+set multiple-cursors
+
+nnoremap , <Esc><Esc>
+xnoremap , <Esc><Esc>
+vnoremap , <Esc><Esc>
+
+xnoremap <Esc>o o
+nnoremap o o
+
+nnoremap d vd
+xnoremap d dv
+
+" Movement
+unmap <Left>
+unmap <Right>
+unmap <Down>
+unmap <Up>
+
+noremap t vt
+noremap T vT
+noremap f vf
+noremap F vF
+
+" Select/extend
+nnoremap e ve
+nnoremap E vE
+nnoremap w vw
+nnoremap W vW
+nnoremap b vb
+nnoremap B vB
+
+noremap X 0V
+xnoremap X <nop>
+
+" Match mode
+noremap ma) <Esc>va(
+noremap ma} <Esc>va{
+noremap ma> <Esc>va<
+noremap ma[ <Esc>va[
+
+noremap mi) <Esc>vi)
+noremap mi{ <Esc>vi}
+noremap mi< <Esc>vi>
+noremap mi[ <Esc>vi]
+
+" Goto mode
+nmap gr <Action>(FindUsages)
+nmap gi <Action>(GotoImplementation)
+nmap gd <Action>(GotoDeclaration)
+nmap gT <Action>(GotoTest)
+nmap gp <Action>(PreviousTab)
+nmap gy <Action>(GotoTypeDeclaration)
+
+" Requires plugin: KJump
+nmap gw <Action>(KJumpAction.Word0)
+xmap gw <Action>(KJumpAction.Word0)
+
+" Requires plugin: String manipulation
+xmap & <Action>(StringManipulation.AlignSelections)
+
+" Selection manipulation
+nmap <A-s> <Action>(EditorAddCaretPerSelectedLine)
+vmap <A-s> <Action>(EditorAddCaretPerSelectedLine)
```

Comment:
Is there a specific reason you're using vmap instead of xmap here?

Reply:
No, it's a leftover from when I started the effort and was still unaware that `x*map` is for visual only and `v*map` affects also command. Fixed now.